### ISSUE TITLE
Event Trigger Lambda Arguments Error

### ISSUE THREAD

**Issue by Alek99** [pynecone-io/pynecone#103]
**Describe the bug**
Lambda need to be able to take in arguments other than ints. Currently non in arguments throw errors for event triggers that use lambdas

**To Reproduce**
``` python
class State(pc.State):
    choices = ["A", "B", "C"]
    
    def set_choice(self, index: int, answer: str):
        self.choices[index] = answer
        self.choices = self.choices

def index():
    return pc.vstack(
        pc.heading(State.choices),
        pc.button(
            "First",
            on_click= lambda: State.set_choice(0, "Changed")
        ),
        pc.button(
            "Second",
            on_click= lambda: State.set_choice(1, "Changed")
        ),
        pc.button(
            "Third",
            on_click= lambda: State.set_choice(2, "Changed")
        )
    )
```
**Expected behavior**
Should set the index of the `choices` to changed depending on button clicked.
**Screenshots**
NA

** Specifics (please complete the following information):**
 - Python Version: 3.10
 - Pynecone Version: 0.1.8
 - OS: Mac
 - Browser (Optional):
 
**Additional context**
None

**Comment by FHU-yezi** [comment]
`lambda _: some_func(index, another)` doesn't work?

**Comment by Alek99** [comment]
No, not if it's anything other than an int, but we are fixing it right now will be included in the 0.1.9 release this week
